Source/Core/Core/PowerPC/PPCSymbolDB.h

```diff
@@ -21,18 +22,20 @@ class PPCSymbolDB : public Common::SymbolDB
   ~PPCSymbolDB() override;
 
   Common::Symbol* AddFunction(u32 start_addr) override;
-  void AddKnownSymbol(u32 startAddr, u32 size, const std::string& name,
+  bool AddKnownSymbol(u32 start_addr, u32 size, u32 virt_addr, u32 file_offset, u32 alignment,
```

Comment:
These additional parameters make the function overly complicated, IMHO. Some of them are also used with a hardcoded 0 parameter and doesn't make more sense.